roles/foreman_installer_devel_scenario/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: 'Install git'
+  package:
+    name: git
+    state: present
+
+- name: 'Copy scenario files'
+  copy:
+    src: "{{ role_path }}/files/{{ item }}"
+    dest: /etc/foreman-installer/scenarios.d
+  with_items:
+    - katello-devel.yaml
+    - katello-devel-answers.yaml
+
+- name: 'Remove current puppet-katello_devel'
+  file:
+    state: absent
+    path: /usr/share/katello-installer-base/modules/katello_devel
+
+- name: 'Clone latest puppet-katello_devel'
+  git:
+    repo: https://github.com/theforeman/puppet-katello_devel
+    dest: /usr/share/katello-installer-base/modules/katello_devel
```

Comment:
It could install the module to ```/usr/share/katello-installer-devel/modules``` and not depend on the installer being present (assuming you add that to ```module_dirs``` but that would break the module PR parameter.

Reply:
I can try this, and hope that the installer RPM install doesn't wipe anything out and plays nice with existing data.

Reply:
If we change the module_prs to the following (possibly in native ansible) then it wouldn't care in which directory it was. Then we can introduce a directory outside of the installer and do it in a single run.

```yaml
- name: "install module PR into installer directory"
  shell: > 
    namespace={{ item.split("/")[0] }}
    mod={{ item.split("/")[1] }}
    pr={{ item.split("/")[2] }}
    for dir in /usr/share/katello-installer-base/modules /usr/share/foreman-installer/modules ; do
      if [ -e $dir/$mod ] ; then
        rm -rf $dir/$mod &&
        git clone https://github.com/$namespace/puppet-$mod $mod &&
        cd $mod &&
        git fetch origin pull/$pr/head:pr &&
        git {{ foreman_installer_module_prs_strategy }} pr
        break
      fi
    done
  with_items: "{{ foreman_installer_module_prs.split(',') }}"
  tags:
    - packages
```

Reply:
I'd also be fine with merging this now and handling this later.

Reply:
So, the non-split method will work but I have to remove the devel module and scenario from the installer RPM first. So I'd say, I'm going to merge this as is. Then remove those bits, then revist.

Reply:
Here is the issue -- https://github.com/theforeman/forklift/issues/652